frontend/src/mocks/handlers.ts

```diff
@@ -50,7 +49,6 @@ export const handlers = mocksToHandlers({
     },
     post: {
         '/e/': () => [200, 'ok'],
-        '/decide/': MOCK_DECIDE,
```

Comment:
Are we safe to remove this? mock service worker treats an API route with no mock as failing (I think)

Reply:
I implemented this mock endpoint for the in app prompts, as it was using feature flags to run, so it shouldn't be needed I believe.